apps/meteor/tests/e2e/e2e-encryption.spec.ts

```diff
@@ -387,3 +392,179 @@ test.describe.serial('e2e-encryption', () => {
 		});
 	});
 });
+
+test.describe.serial('e2ee room setup', () => {
+	let poAccountProfile: AccountProfile;
+	let poHomeChannel: HomeChannel;
+	let e2eePassword: string;
+
+	test.beforeEach(async ({ page }) => {
+		poAccountProfile = new AccountProfile(page);
+		poHomeChannel = new HomeChannel(page);
+	});
+
+	test.beforeAll(async ({ api }) => {
+		const statusCode = (await api.post('/settings/E2E_Enable', { value: true })).status();
+
+		expect(statusCode).toBe(200);
+	});
+
+	test.afterAll(async ({ api }) => {
+		const statusCode = (await api.post('/settings/E2E_Enable', { value: false })).status();
+
+		expect(statusCode).toBe(200);
+	});
+
+	test('expect save password state on encrypted room', async ({ page }) => {
+		await page.goto('/account/security');
+		await poAccountProfile.securityE2EEncryptionSection.click();
+		await poAccountProfile.securityE2EEncryptionResetKeyButton.click();
+
+		await page.locator('role=button[name="Login"]').waitFor();
+
+		await page.reload();
+
+		await page.locator('role=button[name="Login"]').waitFor();
```

Comment:
Is there a reason for that? Clicking on the reset key button should reload and redirect to the login page, besides checking the Login button I think it is worth checking the banner too 
![image](https://github.com/RocketChat/Rocket.Chat/assets/20212776/bf539bd1-1c55-4b01-b32d-3fa2a479d29e)

Reply:
There is(was) already a test implemented for this, but was commented out by Gazzo, [here](https://github.com/RocketChat/Rocket.Chat/blob/49f85478a1ae038ff049ef9c93af2ea0c4bd36c3/apps/meteor/tests/e2e/e2e-encryption.spec.ts#L383-L386)